configure.php

```diff
@@ -386,6 +420,30 @@ function delete_files( string|array $paths ) {
 				}
 			}
 
+			$parent_files = [
+				$parent_folder . '/phpcs.xml',
+				$parent_folder . '/phpcs.xml.dist',
+				$parent_folder . '/.phpcs.xml',
+			];
+
+			if ( file_exists( __DIR__ . '/phpcs.xml' ) ) {
+				foreach ( $parent_files as $parent_file ) {
+					if ( ! file_exists( $parent_file ) ) {
+						continue;
+					}
+
+					if ( confirm( "Do you want to roll up the phpcs configuration to the parent? (This will change the plugin's phpcs configuration to inherit the parent configuration from {$parent_file}.)" ) ) {
+						rollup_phpcs_to_parent(
+							parent_file: '../../' . basename( $parent_file ),
+							local_file: __DIR__ . '/phpcs.xml',
```

Comment:
You may want to consider naming the ruleset `.phpcs.xml` to align with the starter project and starter theme.